util/src/main/scala/Money.scala

```diff
@@ -208,26 +208,7 @@ case class Money private (centAmount: Long, currency: Currency)
     this.centAmount.compare(that.centAmount)
   }
 
-  override def toString: String = {
-    val centAmountString = this.centAmount.toString
-    val formattedString = new StringBuilder()
-    val centAmountLength = centAmountString.length
-    val decimals = this.fractionDigits
-    val missing = decimals - centAmountLength
-    if (missing >= 0) {
-      formattedString.append("0.")
-      for (_ <- 1 to missing)
-        formattedString.append('0')
-    }
-
-    var i = 0
-    centAmountString.iterator.foreach { c =>
-      formattedString.append(c)
-      i += 1
-      if (centAmountLength - i == decimals) formattedString.append(".")
-    }
-    formattedString.result() + " " + this.currency.getCurrencyCode
-  }
+  override def toString: String = Money.toString(centAmount, fractionDigits, currency)
```

Comment:
Should we move that into the trait and remove the `override def toString` completely?

Reply:
I could move it in the BaseMoney trait, but it'll still need to have the parameter "amount". Because Money uses the centAmount and HighPrecisionMoney uses the preciseAmount (which is not on the trait) 
So: 
1. I can move the method inside the trait with 1 parameter (amount)
2. I can move it in the trait and create a private value eg: printedAmount which will be used by the toString

Reply:
I have not seen that they use different variables.
The current approach is good enough.